src/coreComponents/linearAlgebra/utilities/LinearSolverParameters.hpp

```diff
@@ -116,7 +116,11 @@ struct LinearSolverParameters
   {
     real64 relTolerance = 1e-6;       ///< Relative convergence tolerance for iterative solvers
     integer maxIterations = 200;      ///< Max iterations before declaring convergence failure
+#if defined(GEOS_USE_HYPRE_CUDA) || defined(GEOS_USE_HYPRE_HIP)
+    integer maxRestart = 50;          ///< Max number of vectors in Krylov basis before restarting
+#else
```

Comment:
Small suggestion is to try to be consistent between CPU and GPU if possible 
Should simplify the cross comparison and analysis

Reply:
That's a good suggestion. The reason for having different parameters is because each backend (CPU or GPU) have different features. `maxRestart`, for example, can be higher for CPUs because we have more memory available.

If we want "optimal" parameters, it's better to allow their values to differ according to the platform.
If we want easier comparisons, it's better to have same defaults for CPUs and GPUs